code/clue.js

```diff
@@ -1,24 +1,61 @@
 // CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// const mrGreen = {
+//   firstName: "Jacob",
+//   lastName: "Green",
+//   color: "green",
+//   description: "He has a lot of connections",
+//   age: 45,
+//   image: "assets/green.png",
+//   occupation: "Entrepreneur"
+// }
 
-const mrGreen = {
-  firstName: "Jacob",
-  lastName: "Green",
-  color: "green",
-  description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
-  occupation: "Entrepreneur"
+
+
+ function Suspect(firstName, lastName, color, description, age, occupation, image) {
+  this.firstName = firstName;
+  this.lastName  = lastName;
+  this.color = color;
+  this.description = description;
+  this.age = age;
+  this.occupation = occupation;
+  this.image = image;
 }
 
+var mrGreen = new Suspect("Jacob", "Green", "green", "He has a lot of connections", 45, "Entrepreneur", "assets/green.png" );
+var profPlum = new Suspect("Victor", "Plum", "purple", "He is a quick-witted", 40, "Professor", "assets/plum.png" );
+var missScarlet = new Suspect("Cassandra", "Scarlet", "red", "She is young and attractive", 30, "Chef", "assets/scarlet.png"  );
+var mrsPeacock = new Suspect("Eleanor", "Peacock", "blue", "She nearly always maintains her dignity ", 55, "Retired", "assets/peacock.png"  );
+var mrMustard = new Suspect("Colonel ", "Mustard", "yellow", "He is a dangerous military man", 60, "Military man", "assets/mustard.png"  );
+var mrsWhite = new Suspect("Mrs", "White", "lightgrey", "He does not have a lot of connections", 35, "Servant", "assets/white.png"  );
```

Comment:
Do you really need to write you variables with `var`? If possible - try to aim to use `const` or `let` over `var`